rebasehelper/utils.py

```diff
@@ -95,6 +97,36 @@ class ConsoleHelper(object):
 
     """Class for command line interaction with the user."""
 
+    use_colors = False
+
+    @classmethod
+    def should_use_colors(cls, app):
```

Comment:
You are setting `cls.use_colors` attribute where previously there were **return** statements. This means after you set the attribute the execution continues, which can have undesired effects. I think for simplicity and code readability best would be to revert to returning a value and assigning it to `ConsoleHelper.use_colors` outside of the method.